src/main.pl

```diff
@@ -0,0 +1,15 @@
+use 5.36.0;
+use strict;
+use warnings;
```

Comment:
今風なら `use v5.36;` と書くのが良さそうです。
また、`use v5.12;` 以降なら`use strict;`と同等かつ、`use v5.35;`以降なら`use warnings;`も入るので、
```suggestion
use v5.36;
```
これでよいかと

参考: https://perldoc.jp/func/use

> 指定された Perl のバージョンが 5.12 以上の場合、 [use strict](https://perldoc.jp/pod/strict) と同様に、strict 機能がレキシカルに有効になります。 同様に、指定された Perl のバージョンが 5.35.0 以上の場合、 [warnings](https://perldoc.jp/pod/warnings) が有効になります。 後から使った use VERSION は先の use VERSION の全ての振る舞いを上書きするので、 それによって追加された strict, feature, feature を 削除することがあります。 use VERSION は feature.pm, strict.pm, warnings.pm ファイルは 読み込みません。